lua/starfall/libs_sh/file.lua

```diff
@@ -80,10 +80,9 @@ end
 
 --- Creates a directory
 -- @param path Filepath relative to data/sf_filedata/. Cannot contain '..'
--- @return Error message if applicable
 function file_library.createDir ( path )
 	if not SF.Permissions.check( SF.instance.player, path, "file.write" ) then SF.throw( "Insufficient permissions", 2 ) end
 	SF.CheckType( path, "string" )
-	if path:find( "..", 1, true ) then error( "path contains '..'" ) return end
+	if path:find( "..", 1, true ) then SF.throw( "path contains '..'" ) return end
 	return file.CreateDir( "sf_filedata/" .. path )
```

Comment:
Remove the return, since file.CreateDir is silent. Can just be a direct file call, and leave an implicit return.